nephio-ansible-install/roles/docker/tasks/main.yaml

```diff
@@ -0,0 +1,29 @@
+## © 2022 Nokia
+## Licensed under the Apache License 2.0
+## SPDX-License-Identifier: Apache-2.0
+
+---
+- name: ensure repository key is installed
+  become: true
+  apt_key:
+    url: https://download.docker.com/linux/ubuntu/gpg
+    state: present
+
+- name: ensure docker registry is available
+  become: true
+  apt_repository: repo='deb https://download.docker.com/linux/ubuntu bionic stable' state=present
+
+- name: ensure docker and dependencies are installed
+  become: true
+  apt: name=docker-ce update_cache=yes
+
+- name: Add user to docker group
+  become: true
+  shell: usermod -aG docker {{ cloud_user }}
+
+- name: Reset connection so docker group is picked up.
+  meta: reset_connection
+- name: Create the docker network for kind (to make our job easier plumbing up the lot)
```

Comment:
When this task is invoked as part of the install-prereq.yaml playbook, the below error is seen.
I was running the playbook locally using `ansible-playbook playbooks/install-prereq.yaml --connection=local`. I had to log out and log back in; then rerun the playbook again.
It might be a good idea to update the README with instructions that one has to a)log out and log back in b)re-run the playbook again

```
TASK [docker : Create the docker network for kind (to make our job easier plumbing up the lot)] ********************************************************************************************************************************************************************************
fatal: [localhost]: FAILED! => {"changed": false, "msg": "Error connecting: Error while fetching server API version: ('Connection aborted.', PermissionError(13, 'Permission denied'))"}

PLAY RECAP *********************************************************************************************************************************************************************************************************************************************************************
localhost                  : ok=5    changed=4    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0  
```